shell/platform/windows/fixtures/main.dart

```diff
@@ -20,13 +21,24 @@ bool signalBoolReturn() native 'SignalBoolReturn';
 // Notify the native test that the first frame has been scheduled.
 void notifyFirstFrameScheduled() native 'NotifyFirstFrameScheduled';
 
-void main() {
-}
+void main() {}
 
 @pragma('vm:entry-point')
-void customEntrypoint() {
+void hiPlatformChannels() {
+  ui.channelBuffers.setListener('hi',
+      (ByteData? data, ui.PlatformMessageResponseCallback callback) async {
+    ui.PlatformDispatcher.instance.sendPlatformMessage('hi', data,
+        (ByteData? reply) {
+      ui.PlatformDispatcher.instance
+          .sendPlatformMessage('hi', reply, (ByteData? reply) {});
```

Comment:
Is this supposed to be `bye` instead of `hi`? The test seems to be checking for a message starting with `b` too.

Reply:
Yea, it's a bit confusing.  I'm not sure how to make it more clear, "hi" is the name of the channel, not the payload.  Dart just echos the payloads that it gets from the c++ side.